app/controllers/global_issue_templates_controller.rb

```diff
@@ -1,9 +1,8 @@
 # noinspection RubocopInspection
 class GlobalIssueTemplatesController < ApplicationController
   layout 'base'
-  include IssueTemplatesHelper
```

Comment:
helper :issues and include IssuesHelper are the same.
In addition to this, only "helper :issues” is effective.